.travis.yml

```diff
@@ -52,7 +52,8 @@ script:
   - |
       if [[ $TESTS == 'blocks-examples' ]]; then
         cd $TRAVIS_BUILD_DIR
-        git clone https://github.com/mila-udem/blocks-examples.git
+        #git clone https://github.com/mila-udem/blocks-examples.git
+        git clone https://github.com/Thrandis/blocks-examples.git
```

Comment:
You need to clone your branch with corresponding changes (the syntax is `git clone -b my-branch git@github.com:user/myproject.git`).